docs/apache-airflow-providers-amazon/logging/s3-task-handler.rst

```diff
@@ -47,3 +47,86 @@ You can also use `LocalStack <https://localstack.cloud/>`_ to emulate Amazon S3
 To configure it, you must additionally set the endpoint url to point to your local stack.
 You can do this via the Connection Extra ``host`` field.
 For example, ``{"host": "http://localstack:4572"}``
+
+Enabling remote logging for Amazon S3 with AWS IRSA
+'''''''''''''''''''''''''''''''''''''''''''''''''''
```

Comment:
[Using IAM Roles for Service Accounts (IRSA) on EKS](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html#using-iam-roles-for-service-accounts-irsa-on-eks) already contain information about IRSA on EKS in Connection page.

Might be better append general configuration stuff in connection rather than only for S3 Logging?
This configurations might be useful not only for S3 Logging but also in secrets backends and Cloudwatch Logging

Reply:
100% agree on adding IRSA configuration to [this page](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html#using-iam-roles-for-service-accounts-irsa-on-eks).

I will update the page

Reply:
I have updated the aws.rst file as well with general config